src/ontology/ro-edit.owl

```diff
@@ -6147,6 +6149,22 @@ AnnotationAssertion(rdfs:label obo:RO_0016002 "has disease")
 AnnotationAssertion(rdfs:seeAlso obo:RO_0016002 "https://github.com/oborel/obo-relations/issues/478")
 SubObjectPropertyOf(obo:RO_0016002 obo:RO_0016001)
 
+# Object Property: obo:RO_0018001 (is myristoyltransferase activity)
+
+AnnotationAssertion(obo:IAO_0000117 obo:RO_0018001 "Charles Tapley Hoyt"@en)
+AnnotationAssertion(dce:contributor obo:RO_0018001 <https://orcid.org/0000-0003-4423-4370>)
+AnnotationAssertion(rdfs:label obo:RO_0018001 "is myristoyltransferase activity"@en)
+SubObjectPropertyOf(obo:RO_0018001 obo:RO_0002564)
```

Comment:
use rdfs:seeAlso to link the related github issue

Reply:
Added in ae78197